brave-sqs/src/main/java/smartthings/brave/sqs/TracingAmazonSQSClient.java

```diff
@@ -245,10 +245,12 @@ public ReceiveMessageResult receiveMessage(ReceiveMessageRequest receiveMessageR
     // complete in flight one-way spans for all received messages
     for(Message message : result.getMessages()) {
       TraceContextOrSamplingFlags traceContextOrSamplingFlags = extractor.extract(message.getMessageAttributes());
-      TraceContext ctx = traceContextOrSamplingFlags.context();
-      Span oneWay = withEndpoint((ctx != null)
-        ? tracing.tracer().joinSpan(ctx)
-        : tracing.tracer().newTrace(traceContextOrSamplingFlags.samplingFlags()));
+      Span oneWay;
```

Comment:
one way isn't the better choice anymore. this pre-dated messaging model. not sure what's required to fix now vs later